<launch>
  <node pkg="m4atx_battery_monitor" type="m4atx_battery_monitor" name="m4atx" output="screen">
    <rosparam command="load" file="$(find xiroi)/config/m4atx.yaml"/>
    <remap from="battery_status_m4atx" to="sensors/battery"/>
  </node>
</launch>